### DISCUSSION TITLE
Is there a way to add `nofollow` on external links from the config?

### DISCUSSION THREAD

**Discussion by marvindanig** [facebook/docusaurus#8501] [Ideas]
Basically this:

```javascript
const config = {
  title: 'Title here',
  tagline: 'A stack-agnostic title for friends.',
  url: 'https://example.com',
  baseUrl: '/',
  onBrokenLinks: 'throw',
  onBrokenMarkdownLinks: 'warn',
  externalLinks: (['nofollow', 'noopener', 'noreferrer'], { empty object or a list of URLs that are allowed SEO juice.})
  …
  …

```

Apply `nofollow`, `noreferrer`, or `noopener` attributes to any URL on the blog or documentation that is outside of the project domain. Has anyone implemented this for their blog/docs already?

**Comment by slorber** [comment]
We do add noopener noreferrer by default but not nofollow

You can create your own `<AppLink>` component on top of ours

- For React pages: you simply use it
- For md pages: you can make Docusaurus use your custom component for Markdown links by providing `{a: AppLink}` in the MDX components: https://docusaurus.io/docs/2.1.0/markdown-features/react#mdx-component-scope
